Mobile/Android/Templates/471-reliable_uploads_actions_uploads_view.md

```diff
@@ -0,0 +1,64 @@
+|**Multiaccount**||||
+|22| Upload files| 1. Login in two accouts<br> 2. Select files to upload in both accounts | The files are correctly uploaded in each account and shown in the same list of uploads view
+|**Instant Uploads**||||
+|23| Upload a pic (copy)| 1. Enable "Instant picture upload" in settings menu<br> 2. Select "copy" as behaviour<br> 3. Take a pic| The file appears in upload view in "Current" and when it finishes, in "Uploaded". Pic is correctly copied (remains in origin)
+|24| Upload a pic (move)| 1. Enable "Instant picture upload" in settings menu<br> 2. Select "move" as behaviour<br> 3. Take a pic| The file appears in upload view in "Current" and when it finishes, in "Uploaded". Pic is correctly moved (deleted in origin)
+|25| Upload a video (copy)| 1. Enable "Instant video upload" in settings menu<br> 2. Select "copy" as behaviour<br> 3. Take a video| The file appears in upload view in "Current" and when it finishes, in "Uploaded". Video is correctly copied (remains in origin)
+|26| Upload a pic (move)| 1. Enable "Instant video upload" in settings menu<br> 2. Select "move" as behaviour<br> 3. Take a video| The file appears in upload view in "Current" and when it finishes, in "Uploaded". Video is correctly moved (deleted in origin)
+|**Several Actions**||||
+|27| Upload + Share| 1. Tap to upload a file<br> 2. Select File1 to upload<br>3. While it is being uploaded, share a File2 with users/groups | File1 appears in "Uploaded" when the upload is finished. File2 is accesible for the sharees
+|28| Upload + Move| 1. Tap to upload a file<br> 2. Select a File1 to upload<br>3. While it is being uploaded, move a File2 to another location | File1 appears in "Uploaded" when the upload is finished. File2 is correctly moved to its new location (removed in origin)
+|29| Upload + Copy| 1. Tap to upload a file<br> 2. Select a File1 to upload<br>3. While it is being uploaded, copy a File2 | File1 appears in "Uploaded" when the upload is finished. File2 is correctly copied (remains in origin).
+|30| Upload + Delete| 1. Tap to upload a file<br> 2. Select a File1 to upload<br>3. While it is being uploaded, delete a File2 | File1 appears in "Uploaded" when the upload is finished. File2 is correctly removed.
+|31| Upload + Rename| 1. Tap to upload a file<br> 2. Select a File1 to upload<br>3. While it is being uploaded, rename a File2 | File1 appears in "Uploaded" when the upload is finished. File2 is correctly renamed.
+|32| Upload + Send| 1. Tap to upload a file<br> 2. Select a File1 to upload<br>3. While it is being uploaded, send a File2 to another app| File1 appears in "Uploaded" when the upload is finished. File2 is sent correctly.
+|33| Upload + Favourite| 1. Tap to upload a file<br> 2. Select a File1 to upload<br>3. While it is being uploaded, set a File2 as favourite| File1 appears in "Uploaded" when the upload is finished. File2 is now favourite.
```

Comment:
I'd add a test case for favorite files to check that their updates are shown too, something such as: 1.- set a file as favorite, to make it easy a txt file 2.- Open the file and edit it 3.- Changes are uploaded -> expected: the upload is shown on the uploads view